gap/PolygonalComplexes/properties.gi

```diff
@@ -918,4 +918,44 @@ InstallMethod( IsQuadrangular,
 ##
 #######################################
 
+#######################################
+##
+##      Face colouring
+##
+
+InstallMethod( FaceTwoColouring, 
+    "for a polygonal complex",
+	[IsPolygonalComplex],
+	function(complex)
+		local faces,red,blue,f,remfaces,tempfaces,neighbours;
+		if 1 in List(FaceDegreesOfVertices(complex),g->g mod 2) then 
+			Error("Two Colouring does not exist");
+		fi;
+		blue:=[Faces(complex)[1]];
+		red:=[];
+		remfaces:=Difference(Faces(complex),blue);
+		while remfaces <> [] do 
+			tempfaces:=Difference(Faces(complex),Union(blue,red));
```

Comment:
At this place tempfaces and remfaces are the same, so this line is unnecessary if you use in the next line remfaces in the filter.